Recurly/BaseClient.cs

```diff
@@ -67,10 +68,8 @@ public class BaseClient {
 
             // If we have a body, serialize it and add it to the request
             if (body != null) {
-                string json = Json.Serialize(body); 
-                Debug.WriteLine("body: ");
-                Debug.WriteLine(json);
-                request.AddParameter("application/json", json , ParameterType.RequestBody);
+                //request.AddParameter("application/json", json , ParameterType.RequestBody);
```

Comment:
Do we want to leave this commented out in the source code or just go ahead and remove it?

Reply:
gonna remove this one